frontend/src/metabase/lib/urls.js

```diff
@@ -346,3 +346,22 @@ export function extractCollectionId(slug) {
   }
   return extractEntityId(slug);
 }
+
+/*
+ * Will transform a name like `This name has spaces and Uppercases`
+ * into `this-name-has-spaced-and-uppercases`
+ *
+ * then prepend an entity type, say "card" or "dashboard"
+ * plus the passed id.
+ * "
+ */
+export function bookmark({ type, id, name }) {
+  const kebabCaseName = name
```

Comment:
Could `slugg` function be used here instead? Example usage https://github.com/metabase/metabase/blob/570e4a19e9ff0abcb45cc5c442f4e661cc58de5e/frontend/src/metabase/lib/urls.js#L118